the one that adds the vf-core ui colours as tokens

sturobson opened the pull request:
This PR firstly adds the vf-core ui colours to the documentation, giving them their own page, which will close #789.

It also fixes the issue with `vf-ui-color--grey` and `vf-ui-color--grey--light` where they needed to be swapped, which will close #788.

They are swapped in the docs and in the CSS for components (I think I got them all).

khawkins98 commented:
For the `vf-ui-color--grey--light` and `vf-ui-color--grey` mismatch in #788, I think the problem is in the design tokens; https://visual-framework.github.io/vf-core/components/detail/vf-utility-classes--background.html:

![image](https://user-images.githubusercontent.com/928100/79028007-c0e49600-7b8e-11ea-9370-5969d652768f.png)

If I'm following this PR, it swaps the usage of the design tokens but `grey--light` is still darker than `grey`?

sturobson commented:
I’ll have another look at this - I thought some of the components benefited from the change too … Will revise and push any changes to the PR on Monday

sturobson committed:
starts again and adds ui colours

I misunderstood the requirements so this latest commit fixes that by just: creating the ui colours documentation, updates the ui grey light and grey lightest

sturobson commented:
I got myself in a right not, a `git revert` didn't work (locally) as I'd expected so I reset the branch to `develop` and started again (luckily atom keeps files opened so a quick `cmd + z` and `cmd + s` and I think we're where this PR needed to be.

khawkins98 reviewed (approved):
👍 looks good
